src/geocoder/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.SearchAndCompare.Api;
+using GovUk.Education.SearchAndCompare.Api.DatabaseAccess;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DependencyCollector;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+
+namespace GovUk.Education.SearchAndCompare.Geocoder
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var configuration = GetConfiguration();
+
+            var logger = new LoggerConfiguration()
+                .ReadFrom.Configuration(configuration)
+                .WriteTo
+                .ApplicationInsightsTraces(configuration["APPINSIGHTS_INSTRUMENTATIONKEY"])
+                .CreateLogger();
+
+            logger.Information("Geocoder started.");
+
+            var res = RequestLocations(configuration, logger).Result;
```

Comment:
Instead of passing in config directly

use something similar to 

https://github.com/DFE-Digital/manage-courses-ucas-importer/blob/fa2c92e0847eba7ce55ecc55617dad9d73c6f5a4/src/importer/Program.cs#L33

Its zero funning following the code to find what is needed to be set